node/src/Transaction.ts

```diff
@@ -1972,7 +1973,8 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
      *
      * @param key - The key of the stream.
      * @param values - field-value pairs to be added to the entry.
-     * @returns The id of the added entry, or `null` if `options.makeStream` is set to `false` and no stream with the matching `key` exists.
+     *
+     * Command Response - The id of the added entry, or `null` if `options.makeStream` is set to `false` and no stream with the matching `key` exists.
```

Comment:
Add param options since you're here.
Do we need to update it in base client too?

Reply:
Do we need to update other clients too?

Reply:
Java and Node have options included in docs here.